docs/handbook/triggers.mdx

```diff
@@ -0,0 +1,180 @@
+
+```json
+{
+  "trigger": {
+    "type": "evm",
+    "config": {
+      "address": "0x1234...",     // Contract address to monitor
+      "event_hash": "0x...",      // Event hash (32 bytes)
+      "chain_name": "ethereum"    // Chain to monitor
+    }
+  }
+}
+```
+
+### 2. Cosmos Event Trigger
+
+This trigger monitors events emitted by Cosmos smart contracts, executing your component when a matching event type is detected from the specified contract address.
+
+```json
+{
+  "trigger": {
+    "type": "cosmos",
+    "config": {
+      "address": "cosmos1...",    // Bech32 contract address
+      "event_type": "transfer",   // Event type to watch
+      "chain_name": "cosmoshub-4" // Chain to monitor set in wavs.toml
+    }
+  }
+}
+```
+
+### 3. Cron Trigger
+
+Executes your component on a schedule defined by a cron expression, with optional start and end times to control the execution window.
+
+```json
+{
+  "trigger": {
+    "type": "cron",
+    "config": {
+      "schedule": "*/45 * * * * *",  // Cron expression
+      "start_time": 1704067200000000000,  // Optional start time in nanoseconds since Unix epoch (2024-01-01T00:00:00Z)
+      "end_time": 1735689599000000000     // Optional end time in nanoseconds since Unix epoch (2024-12-31T23:59:59Z)
+    }3
+  }
+}
+
+// Cron Expression Format:
+// * * * * * *
+// │ │ │ │ │ │
+// │ │ │ │ │ └── Day of week (0-6, where 0 is Sunday)
+// │ │ │ │ └──── Month (1-12)
+// │ │ │ └────── Day of month (1-31)
+// │ │ └──────── Hour (0-23)
+// │ └────────── Minute (0-59)
+// └──────────── Second (0-59)
+//
+// Each field can be:
+// - A number: `5` (exact time)
+// - A range: `1-5` (1 through 5)
+// - A list: `1,3,5` (1, 3, and 5)
+// - A step: `*/5` (every 5 units)
+// - An asterisk: `*` (every unit)
+//
+// Common examples:
+// - `*/45 * * * * *` - Every 45 seconds
+// - `0 */5 * * * *` - Every 5 minutes
+// - `0 0 * * * *` - Every hour
+// - `0 0 12 * * *` - Every day at noon
+// - `0 0 12 1 * *` - Noon on the first day of every month
+// - `0 0 12 * * 1` - Noon every Monday
+```
+
+### 4. Block Trigger
+
+Executes your component at regular block intervals on a specified EVM or Cosmos chain, useful for chain-specific operations that need to run periodically.
+
+```json
+{
+  "trigger": {
+    "type": "block",
+    "config": {
+      "chain_name": "ethereum",  // Must match exactly what's in wavs.toml
+      "n_blocks": 10            // Must be a non-zero number
+    }
+  }
+}
+```
+
+The `chain-name` in your block trigger configuration is set in `wavs.toml` under the `[default.chains.evm.<chain_name>]` or `[default.chains.cosmos.<chain_name>]` section. It also needs to be added to the `active_trigger_chains` list in `wavs.toml`.
+
+```toml
+[wavs]
+active_trigger_chains = ["<chain_name>"]
+```
+
+
+### 5. Manual Trigger
+The Manual Trigger is a testing-only trigger that allows manual execution of your component, not intended for production use.
+
+```json
+{
+  "trigger": {
+    "type": "manual"
```

Comment:
i dont think you actually do this? (like it's not exposed). The `wavs-cli exec` just has this internally to magically do it for testing. you would not set manual in the service config unless seen elsewhere / told from david

Reply:
Will remove.